src/__tests__/utils/getTag.ts

```diff
@@ -0,0 +1,16 @@
+
+/**
+ * Возвращает строковое представление типа значения `value`.
+ * Если значение `value` равно `null` или `undefined`, возвращает соответствующую строку.
+ * В противном случае возвращает результат вызова метода `toString` для `value`.
+ *
+ * @param {any} value - Значение для получения его тега.
+ * @returns {string} Возвращает строковое представление типа значения.
+ */
+export function getTag(value: any) {
```

Comment:
здесь видимо можно безопасно добавить возвращаемый тип:
```suggestion
export function getTag(value: any): string {
```